packages/node-opcua-end2end-test/test/end_to_end/test_e2e_1170.ts

```diff
@@ -0,0 +1,79 @@
+const async = require("async");
+const should = require("should");
+const opcua = require("node-opcua");
+const OPCUAClient = opcua.OPCUAClient;
+const UserTokenPolicy = opcua.UserTokenPolicy;
+const { perform_operation_on_client_session } = require("../../test_helpers/perform_operation_on_client_session");
+// require("node-opcua-service-session").UserNameIdentityToken;
+
+const describe = require("node-opcua-leak-detector").describeWithLeakDetector;
+describe("Testing bug #1170", function() {
+
+    const port = 1170;
```

Comment:
Please make sure that port is not used anywhere else in other unit test ... We run unit tests in parallel and all severs must have a different port.

Reply:
This port is not used by other tests